LocalMultiplayerAgent/MultiplayerSettings.json

```diff
@@ -2,8 +2,13 @@
     "RunContainer": false,
     "OutputFolder": "",
     "NumHeartBeatsForActivateResponse": 10,
+    "NumHeartBeatsForMaintenanceEventResponse": 0, // a value < 1 will disable the maintenance event
```

Comment:
I'd say we start with a negative number, to disable this by default, since that's the way LMA works now (not sending maintenance events)

Reply:
0 is disabled already. Heartbeat count starts at 1